boards/arm/nrf52840_pca10059/doc/index.rst

```diff
@@ -107,82 +107,189 @@ Programming and Debugging
 
 Applications for the ``nrf52840_pca10059`` board configuration can be
 built in the usual way (see :ref:`build_an_application` for more details).
-There are two ways to program the board, with the DFU tool or with an external
-debugger chip.
 
-Flashing with a device firmware update (DFU)
-============================================
+Flashing
+========
 
-The board is factory-programmed with Nordic's bootloader from Nordic's nRF5 SDK.
-This section covers the steps required to program the board with a Zephyr
-application using Nordic's DFU tool nrfutil.
+The board supports the following programming options:
 
-Follow the instructions on `nrfutil GitHub`_ to install the necessary software.
+1. Using the built-in bootloader only
+2. Using MCUboot in serial recovery mode
+3. Using an external :ref:`debug probe <debug-probes>`
 
-Compile a Zephyr application as usual,
-here is an example for the :ref:`blinky-sample` application.
+These instructions use the :ref:`west <west>` tool and assume you are in the
+root directory of your :term:`west installation`.
 
-.. zephyr-app-commands::
-   :zephyr-app: samples/basic/blinky
-   :board: nrf52840_pca10059
-   :goals: build
+Option 1: Using the Built-In Bootloader Only
+--------------------------------------------
+
+The board is factory-programmed with Nordic's bootloader from Nordic's nRF5
+SDK. With this option, you'll use Nordic's `nrfutil`_ program to create
+firmware packages supported by this bootloader and flash them to the
+device. Make sure ``nrfutil`` is installed before proceeding.
+
+#. Reset the board into the Nordic bootloader by pressing the RESET button.
```

Comment:
Will the board stay in bootloader mode long enough to do the rest of these steps (will it time out)?  Or should this step be moved later in the step list?  It seems like you'd want to compile and package the software, then reset the board right before you flash.  I think.

Reply:
The board stays in bootloader mode forever (unlike `arduino_101` and boards which behave like it).